src/transformers/commands/lfs.py

```diff
@@ -0,0 +1,219 @@
+"""
+Implementation of a custom transfer agent for the transfer type "multipart" for git-lfs.
+
+Inspired by: github.com/cbartz/git-lfs-swift-transfer-agent/blob/master/git_lfs_swift_transfer.py
+
+Spec is: github.com/git-lfs/git-lfs/blob/master/docs/custom-transfers.md
+
+
+To launch debugger while developing:
+
+``` [lfs "customtransfer.multipart"]
+
+path = /path/to/transformers/.env/bin/python
+
+args = -m debugpy --listen 5678 --wait-for-client /path/to/transformers/src/transformers/commands/transformers_cli.py
+lfs-multipart-upload ```
+"""
+
+import json
+import os
+import subprocess
+import sys
+from argparse import ArgumentParser
+from contextlib import AbstractContextManager
+from typing import Dict, List, Optional
+
+import requests
+from transformers.commands import BaseTransformersCLICommand
+
+from ..utils import logging
+
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+LFS_MULTIPART_UPLOAD_COMMAND = "lfs-multipart-upload"
+
+
+class LfsCommands(BaseTransformersCLICommand):
+    # Implementation of a custom transfer agent for the transfer type "multipart" for git-lfs.
+    # This lets users upload large files >5GB 🔥.
+    # Spec for LFS custom transfer agent is: https://github.com/git-lfs/git-lfs/blob/master/docs/custom-transfers.md
+    #
+    # This introduces two commands to the CLI:
+    #
+    #     transformers-cli lfs-enable-largefiles
+    #
+    # Do this once per model repo where you want to push >5GB files.
+    # It's documented in the error message you get if you just try to git push a 5GB file
```

Comment:
I think it would be more useful to have the example here that was posted in the PR description: 
Something like.
```
Example::

  $ git clone https://huggingface.co/t5-3b
  $ cd t5-3b
  $ git remote set-url origin https://huggingface.co/$USER/t5-3b-clone
  $ # ^ After having created this model repo in your account
  $ transformers-cli lfs-enable-largefiles
  $ git push
```